spec/user_interface_spec.rb

```diff
@@ -126,6 +129,53 @@
     end
   end
 
+  describe '#add_another_contact?' do
+    it 'prints prompt to user' do
+      input = StringIO.new('y')
+      ui = described_class.new(input, output)
+
+      ui.add_another_contact?
+
+      expect(output.string).to include(described_class::ANOTHER_CONTACT_PROMPT)
+    end
+
+    it 'returns true if user input is y' do
```

Comment:
The user input will only be y in English. Also y is an implementation details, because even in English, you may have to type yes instead.

To document this part of the codebase, this would read better:

returns true if user wants to add another contact
        
returns false if user does not want to add another contact